Code/src/test/java/nl/utwente/group10/haskell/type/ConstTTest.java

```diff
@@ -0,0 +1,24 @@
+package nl.utwente.group10.haskell.type;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class ConstTTest {
+    @Test
+    public final void testToHaskellType() {
+        final ConstT integer = new ConstT("Integer");
+        final ConstT integerWithArg = new ConstT("Integer", integer);
```

Comment:
`Maybe Int` might be a good example, i.e. what `lookup 10 [(Int, Int)]` gives you.